src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,12 @@
+package codeu.chat.common;
+import codeu.chat.util.Time;
+
+public final class ServerInfo {
+	public final Time startTime;
+	public ServerInfo() {
+		this.startTime = Time.now();
+	}
+	public ServerInfo(Time startTime) {
```

Comment:
There is a reason this constructor was also provided (for testing purposes where you want to set a custom start time).

Will be nice to add a junit test which covers this and other changes in this PR.